src/styles/utils/_normalize.scss

```diff
@@ -0,0 +1,11 @@
+html {
+  font-family: Roboto, sans-serif;
```

Comment:
Mb, do we need to link this font family?)

Reply:
> Mb, do we need to link this font family?)

Like "font-family: $font-family"?